packages/hash/api/src/graphql/typeDefs/page.typedef.ts

```diff
@@ -261,6 +273,38 @@ export const pageTypedef = gql`
     newEntityEntityId: ID!
   }
 
+  """
+  Create an entity, which you can then reference in other actions, such as a InsertBlockAction
+  """
+  input CreateEntityAction {
+    entity: EntityDefinition!
+    entityPlaceholderId: ID
```

Comment:
`entityPlaceholderId` is nullable in the GQL types but used in the placeholder map of type `Map<string, string>`. is it meant to be nullable?

Reply:
`isPlaceholderId` is used to check if this is a valid placeholder before setting it to the map. It's ignored if it isn't. You only need to set an `entityPlaceholderId` if you want to reference this in another part of the mutation actions, or if you want to get the entityId out of the result. It's supposed to be nullable.